Arrays/Problem19/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace Problem19
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+
+            /*
+               Write a program, which finds all prime numbers in the range
+               [1 … 1,000,000].
+             */
+            const int max = 100000;
+
+            bool[] primes = new bool[max];
+            primes = Array.ConvertAll(primes, prime => prime = true);
+
+            for (int i = 2; i < Math.Sqrt(max); i++)
```

Comment:
Leave comments when using some specific logic like ("The sieve of Erathostenes")